### ISSUE TITLE
Loader - Indicate live regions for dynamically changing content - (2036570575)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-design-system#5587]
### Module: 
 20a Loader - also includes #6534

### Violation: 
 Indicate live regions for dynamically changing content 

![image](https://user-images.githubusercontent.com/5023024/202742123-a9aea174-d511-4571-bf95-a9e2595ae604.png)

### WCAG Reference: 
- [1.3.1 Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html) 

### Instance ID: 
 2036570575 

### Severity: 
 9 

### Description: 
 [Pattern: ARIA live region - Missing for loading indicator]

[Issue]
The loading indicator is not announced for screen reader users. Even though the role=progressbar is provided the loading message is not announced

### Occurrences
**Ensure all the following components are met when providing context through an `aria-live` region.**
- [x] 20a Loader
- [x] 23a Panel
- [ ] 25b Scrim

In 6534:
~- [ ] 09a Button~
~- [ ] 09b Split Button~
~- [ ] 14a Floating Action Button (FAB)~
~- [ ] 15 Inline Editable~
~- [ ] 18b Card~

[User Impact]
Screen reader users will be unaware when the page is in a loading state.

[Code Reference] 
 ```
<calcite-loader class="loading-in" aria-label="Loading" id="1179fac2-76b9-ac8d-9db7-4062b0d421a0" role="progressbar" active="" inline="" scale="m" calcite-hydrated="">
<div class="loader__svgs">
<svg class="loader__svg loader__svg--1" viewBox="0 0 16 16">
(...)
</svg>
</div>
</calcite-loader>
 ``` 

### Note: 
 [Suggestion]
Indicate live regions for dynamically changing content. Live regions can be created by adding a role attribute set to "log", "status", "alert", "progressbar", "marquee", or "timer" as appropriate. Alternatively, custom behavior can be created by using the aria-live, aria-atomic, and aria-relevant attributes. Text injected into this live region element will be announced by screen readers.

Importantly, the element with the ARIA live attributes must be available when the page loads. Otherwise, many screen readers will not detect updates to the element. Additionally, the element must be empty on page load unless an immediate screen reader announcement is desired.

To create such a region:
- Ensure that a visually hidden container ``` <div> ``` or ``` <span> ``` is present when the page loads. - Add aria-live="polite" attribute to this ``` <div> ``` or ``` <span> ```
- When the page is in a loading state, use JavaScript to inject text into this ``` <div aria-live="polite"> ``` that describes the loading behavior (e.g., "Loading"). 

[Compliant Code Example] 
 ```
N/A
 ```
 

### Media Type: 
 Live Regions 

### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by geospatialem** [comment]
Could be coupled with effort from https://github.com/Esri/calcite-components/issues/5631

**Comment by geospatialem** [comment]
All of the `loader`s are looking good! 👍🏻 

It does seem like Scrim, while unavailable to mouse users, is still available to keyboard and AT users. So context isn't currently provided that the Scrim is present, and a user can select any content hidden behind the component via keyboard. For instance, in https://codepen.io/geospatialem/pen/YzdKqwN. cc @anveshmekala

**Comment by geospatialem** [comment]
Team discussion will be held in the near future to determine next steps - if documentation updates are needed and its up to the developer to make content inaccessible to keyboards, or if we should tackle effort on the `scrim` component via a new issue.

**Comment by anveshmekala** [comment]
can we close this one as discussed @geospatialem

**Comment by geospatialem** [comment]
Since `scrim` is visual component, making it accessible is bit complex considering how users leverages the component for specific use case and the accessibility state should be coming from the component which is using `scrim` as a slot. Adjusting the assignment and change to a doc update to update the docs page stating accessibility is not included.

**Comment by geospatialem** [comment]
> Since `scrim` is visual component, making it accessible is bit complex considering how users leverages the component for specific use case and the accessibility state should be coming from the component which is using `scrim` as a slot. Adjusting the assignment and change to a doc update to update the docs page stating accessibility is not included.

Added an "Accessibility" section to the [Scrim component](https://developers.arcgis.com/calcite-design-system/components/scrim/) page, due for release tomorrow, 10/26.

**Comment by geospatialem** [comment]
Verified on the Scrim component page under the new [Accessibility](https://developers.arcgis.com/calcite-design-system/components/scrim/#accessibility) section.
